js/components/agency_component_preview.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import AgencyComponentProcessingTime from './agency_component_processing_time';
+import FoiaPersonnel from './foia_personnel';
+import FoiaSubmissionAddress from './foia_submission_address';
+import PrettyUrl from './pretty_url';
+import { AgencyComponent } from '../models';
+
+
+function AgencyComponentPreview({ agencyComponent }) {
+  const description = AgencyComponent.agencyMission(agencyComponent);
+  const requestUrl = `/request/agency-component/${agencyComponent.id}/`;
+  debugger;
+  return (
+    <div className="agency-preview usa-grid-full">
+      <div className="usa-width-one-whole">
+        <h2>{agencyComponent.agency.name}</h2>
+        <h3>{agencyComponent.title}</h3>
+      </div>
+      <div className="usa-width-one-half">
+        { description &&
+          <div>
+            <h4>Agency mission</h4>
+            <p>{description}</p>
+          </div>
+        }
+
+        { agencyComponent.request_time_stats &&
+          <div>
+            <h4>Median processing time (estimates)</h4>
```

Comment:
@RyanSibley do we actually want to refer to these as "estimates"? I feel like we want to steer people away from thinking of these as a prediction.